sky/cli.py

```diff
@@ -2938,8 +2938,16 @@ def show_gpus(
     To show all accelerators, including less common ones and their detailed
     information, use ``sky show-gpus --all``.
 
-    NOTE: If region is not specified, the price displayed for each instance type
-    is the lowest across all regions for both on-demand and spot instances.
+    Definitions of certain fields:
+
+    * ``DEVICE_MEM``: Memory of a single device; does not depend on the device
+      count of the instance (VM).
+
+    * ``HOST_MEM``: Memory of the host instance (VM).
```

Comment:
nit: Do we need `` here?

Reply:
Yea, this doesn't look good on `-h` but is what's needed for rst/sphinx docs.